about.html

```diff
@@ -0,0 +1,102 @@
+    <title>CREATIVE CITY SUMMIT</title>
+</head>
+
+<body>
+    <header>
+        <nav class="upper-nav">
+            <ul class="upper-ul">
+                <li><a href="#"><img src="./images/facebook.png" alt="facebookIcon"></a></li>
+                <li><a href="#"><img src="./images/twitter.png" alt="twitter"></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="https://www.google.com/">Log out</a></li>
+            </ul>
+        </nav>
+        <nav class="sec-navbar">
+            <span class="menu" id="menu"><i class="fas fa-bars"></i></span>
+            <img class="logo" src="./images/logo.png" alt="logo">
+            <ul class="second-list" id="mobile-menu">
+                <li><i class="fas fa-times close-btn" id="close-btn"></i></li>
+                <li><a href="index.html" class="nav-list" id="about-link">Home</a></li>
+                <li><a href="#" class="nav-list">Program</a></li>
+                <li><a href="#" class="nav-list">Join</a></li>
+                <li><a href="#" class="nav-list">Sponsor</a></li>
+                <li><a href="#" class="button nav-list">CC Campaign</a></li>
+            </ul>
+        </nav>
+    </header>
+    </main>
+    <main class="main-container about-container">
+        <div class="center-about">
+            <h1 class="top-h1-about">Hello! sharing world</h1>
+            <h2 class="main-sec-heading-about">CREATIVE CITY SUMMIT 2015</h1>
+            <p class="main-par-about">
+                Connecting with sector colleagues is an essential part of developing culture in Canada. Every year, delegates representing culture work across the country gather to share, learn, and grow..
+                Connecting with sector colleagues is an essential part of developing culture in Canada. Every year, delegates representing culture work across the country gather to share, learn, and grow..
+            </p>
+            <p class="contact-us">
+                Do you have other questions about the CC Global Summit 2015! </p>
+            <a href="#" class="email-id">ccsummit2015@cccanada.org </a>
+        </div>
+    </main>
+    <section class="section-2">
+        <h2 class="about-h2">Creative city Global Summit 2015 Logo</h2>
+        <div class="orange-underline"></div>
+        <p class="about-par">
+            Through the logo contest, which was held from June 8th to July 7th, the winning design was determined
+            through online voting and evaluation by judges.
+            The logo of CC Global Summit 2015, which was optimized for use on the homepage and promotional materials
+            after consultation with the winning designer, is as follows.
+        </p>
+        <img src="./images/about_logo.png" alt="logo-2" class="about-logo">
+    </section>
+
+    <section class="section-3">
+        <h2 class="about-h2">View Past Global Summits</h2>
```

Comment:
- [OPTIONAL] Consider giving this element more `margin-top` to make it similar to the original design.